cdap-ui/app/features/mapreduce/mapreduce.less

```diff
@@ -27,4 +27,9 @@ body.state-mapreduce{
       border-right: 1px solid black;
     }
   }
+  .btn.btn-default.no-border{
+    &:focus{
+      outline: none;
+    }
+  }
```

Comment:
Thought this is only specific for program types these should be in respective programs. Was not too sure how to create a program level styling.